velox/functions/prestosql/tests/ComparisonsTest.cpp

```diff
@@ -414,6 +459,65 @@ TEST_F(ComparisonsTest, eqArray) {
       std::nullopt);
 }
 
+TEST_F(ComparisonsTest, neqMap) {
```

Comment:
Since the tests for neqMap is very similar to eqMap, could we generalize them to avoid code duplication? (same for neqArray)

Reply:
I generalized them to a single test where we can pass the type of expression to use a single function.